components/infra-proxy-service/habitat/plan.sh

```diff
@@ -0,0 +1,46 @@
+#shellcheck disable=SC2034
+#shellcheck disable=SC2154
+
+pkg_name=infra-proxy-service
+pkg_description="Automate infra views"
+pkg_origin=chef
+pkg_version="0.1.0"
+pkg_maintainer="Chef Software Inc. <support@chef.io>"
+pkg_license=('Chef-MLSA')
+pkg_upstream_url="http://github.com/chef/automate/components/infra-proxy-service"
+pkg_deps=(
+  core/bash
+  "${local_platform_tools_origin:-chef}/automate-platform-tools"
+  chef/mlsa
+)
+pkg_exports=(
+  [port]=service.port # default service is grpc
```

Comment:
I think you might want to add a `[host]=service.host` to the pkg_exports like below. 
```
pkg_exports=(
  [port]=service.port
  [host]=service.host
)
```
This allows other services to know how to connect to this service through localhost or the IP address.

Reply:
Yes, thanks for pointing it out.